types/src/transaction/authenticator.rs

```diff
@@ -358,6 +358,9 @@ impl TransactionAuthenticator {
                 AccountAuthenticator::MultiKey { authenticator } => {
                     single_key_authenticators.extend(authenticator.to_single_key_authenticators()?);
                 },
+                AccountAuthenticator::NoAccountAuthenticator => {
+                    //  This case adds no single key authenticators to the vector.
+                },
```

Comment:
Does this get rejected by normal requests?

Reply:
The use of `NoAccountAuthenticator` should result in a failure during the signature verification of the transaction execution:
```
    pub fn verify<T: Serialize + CryptoHash>(&self, message: &T) -> Result<()> {
        match self {
            ...
            Self::NoAccountAuthenticator => bail!("No signature to verify."),
        }
    }
```
(https://github.com/aptos-labs/aptos-core/pull/13714/files#diff-770742fc2aa984d2ad09f991d7dd190d3e0888189a87e3d3d25768d6d1939dbeR568)